<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eyestack::Com_DLP::DLPConfigUi</class>
 <widget class="QWidget" name="Eyestack::Com_DLP::DLPConfigUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1131</width>
     <height>701</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>内部投影</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1091</width>
       <height>651</height>
      </rect>
     </property>
     <widget class="QPushButton" name="ProgramButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>530</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Program And LoadPattern</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RunButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>600</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Run Once</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PauseButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>600</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StopButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>600</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SaveButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>490</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Patterns To File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LoadButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>550</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Pattern From File</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>271</width>
        <height>28</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="minimumContentsLength">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Please Add a Pattern Set</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Add 1-bit Horizontal Pattern</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Add 1-bit Vertical Pattern</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Add 8-bit Horizontal Pattern</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Add 8-bit Vertical Pattern</string>
       </property>
      </item>
     </widget>
     <widget class="QTableWidget" name="ConfigTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>981</width>
        <height>181</height>
       </rect>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>4</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>35</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>160</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>38</number>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>PatternSet</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ilumination</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>图案数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>前曝光时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>曝光时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>后曝光时间</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>1.Create A PatternSet:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>2.Add 1D Pattern</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3.Configuation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>4.Program Flash</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>570</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>5.Display</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ImageButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>181</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Add A Pattern</string>
      </property>
     </widget>
     <widget class="QListView" name="Pattern">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>140</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
     </widget>
     <widget class="QListWidget" name="PatternSet">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>80</y>
        <width>92</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DeleteButton">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>40</y>
        <width>92</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DumpPatternSet">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>220</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>导入PatternSet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ApplyButton">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>350</y>
        <width>92</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>应用</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>棋盘格投影</string>
    </attribute>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>网格</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>棋盘格</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
